src/pages/relicans.js

```diff
@@ -4,344 +4,115 @@ import SEO from '../components/Seo';
+                it. Our entire platform is designed to make their lives easier.
+                We’re here to help you write better software and we’re very
+                proud to announce that we’re launching a developer relations
+                team to do just that.
               </p>
               <p>
-                You're going to be hearing a lot from us in the coming months as we join you out there on the internet, here's a quick preview of what we'll be up to:
+                You're going to be hearing a lot from us in the coming months as
+                we join you out there on the internet, here's a quick preview of
+                what we'll be up to:
               </p>
               <p>
                 <ul>
                   <li>
-                    Producing educational and inspirational content for anyone who writes code, whether in development, site reliability engineering, or IT operations.
+                    Producing educational and inspirational content for anyone
+                    who writes code, whether in development, site reliability
+                    engineering, or IT operations.
                   </li>
                   <li>
-                    Partnering with you as you explore the world of software, not just on our platform but with all of the languages, frameworks, and tools you love.
+                    Partnering with you as you explore the world of software,
+                    not just on our platform but with all of the languages,
+                    frameworks, and tools you love.
                   </li>
                   <li>
-                    Inviting you to hang out with us for gaming, writing code, playing music, and just having fun.
+                    Inviting you to hang out with us for gaming, writing code,
+                    playing music, and just having fun.
                   </li>
                 </ul>
               </p>
               <p>
-                You can find news of our efforts here on this page or on our Twitter account: <a href="https://twitter.com/therelicans">@therelicans</a>.
+                You can find news of our efforts here on this page or on our
+                Twitter account:{' '}
+                <a href="https://twitter.com/therelicans">@therelicans</a>.
               </p>
             </div>
-            <iframe width="560" height="315" src="https://www.youtube.com/embed/OlW3g631ero" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+            <iframe
+              css={css`
+                max-width: 560px;
+                iframe {
+                  width: 100%;
+                  height: 315px;
+                }
+              `}
+              width="560"
+              title="Relicans Page Video"
+              height="315"
+              src="https://www.youtube.com/embed/OlW3g631ero"
```

Comment:
It looks like this is the same URL as the `Video` component generates: https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/src/components/Video.js#L8

By all means, that `id` should work. What error were you seeing with the `Video` component when using it? I'd much prefer that if we can since it has all of the styles nicely contained in it.

Reply:
@jerelmiller I totally agree. When I was doing it, the video would not appear. I'm not sure why because I was getting no errors

Reply:
The only thing I can think of is that for some reason the size of the video was `0x0`, so even if it was being rendered, it wasn't seen because it had no width/height. That would be my guess for why that was happening. We can look into it later. For now this is fine, though my guess is this doesn't scale with mobile well.